ibm/service/kms/resource_ibm_kms_key_policies_test.go

```diff
@@ -55,18 +55,18 @@ func TestAccIBMKMSKeyPolicy_basic_check_enable(t *testing.T) {
 				Config: testAccCheckIBMKmsKeyPolicyStandardConfigCheckEnable(instanceName, keyName, rotationEnable, rotation_interval, dual_auth_delete),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("ibm_kms_key.test", "key_name", keyName),
-					resource.TestCheckResourceAttr("ibm_kms_key.test", "policies.0.rotation.0.enabled", "true"),
-					resource.TestCheckResourceAttr("ibm_kms_key.test", "policies.0.rotation.0.interval_month", "3"),
-					resource.TestCheckResourceAttr("ibm_kms_key.test", "policies.0.dual_auth_delete.0.enabled", "false"),
+					resource.TestCheckResourceAttr("data.ibm_kms_key.test2", "keys.0.policies.0.rotation.0.enabled", "true"),
+					resource.TestCheckResourceAttr("data.ibm_kms_key.test2", "keys.0.policies.0.rotation.0.interval_month", "3"),
+					resource.TestCheckResourceAttr("data.ibm_kms_key.test2", "keys.0.policies.0.dual_auth_delete.0.enabled", "false"),
```

Comment:
Do we have test case that updates both interval month and enabled at the same time?

Reply:
**Added a separate test case will includes updated value of  enabled as `false` and new interval_month as `5`.** 
<img width="783" alt="Screenshot 2022-11-03 at 10 40 09 PM" src="https://user-images.githubusercontent.com/25618813/199788952-2bc5a875-625f-4ef0-bbfd-1a9e9aa44a34.png">